pkg/controllers/taint/clustertaintpolicy_controller.go

```diff
@@ -58,18 +59,18 @@ type ClusterTaintPolicyController struct {
 // The Controller will requeue the Request to be processed again if an error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (c *ClusterTaintPolicyController) Reconcile(ctx context.Context, req controllerruntime.Request) (controllerruntime.Result, error) {
-	klog.V(4).Infof("Reconciling Cluster %s.", req.Name)
+	klog.V(4).InfoS("Reconciling Cluster", "cluster", req.Name)
 
 	clusterObj := &clusterv1alpha1.Cluster{}
 	if err := c.Client.Get(ctx, req.NamespacedName, clusterObj); err != nil {
-		klog.Errorf("Failed to get Cluster(%s), error: %v", req.Name, err)
+		klog.ErrorS(err, "Failed to get Cluster", "cluster", req.Name)
 		return controllerruntime.Result{}, client.IgnoreNotFound(err)
 	}
 
 	clusterTaintPolicyList := &policyv1alpha1.ClusterTaintPolicyList{}
 	listOption := &client.ListOptions{UnsafeDisableDeepCopy: ptr.To(true)}
 	if err := c.Client.List(ctx, clusterTaintPolicyList, listOption); err != nil {
-		klog.Errorf("Failed to list ClusterTaintPolicy, error: %v", err)
+		klog.ErrorS(err, "Failed to list ClusterTaintPolicy", "clusterTaintPolicyList", listOption)
```

Comment:
I think this is fine without printing the list options. Please remove that part.